### ISSUE TITLE
[feature request] use context

### ISSUE THREAD

**Issue by trebmuh** [lutostag/webextension-formsave#21]
**Scenario**: I'm "welcoming-administrator" on a forum. To achieve that, I'm sending a welcoming private message to each new registered user. For efficiency, I'm using a "base-message" that I'm modifying to fit each new comer.

**Feature request**: it'd be great if, when I'm on the form-page, I would be able to right-click and then call back this base-message from a choice of "the last messages send from this form", rather than having to click on the formsave icon, then search amongst the multiple forms saved there.

**Info**: Firefox 60.2

**Comment by lutostag** [comment]
Good idea. I think this context menu capability would work nicely for a lot of use cases. Definitely something to do in the future.
